tests/src/test/java/io/streamnative/pulsar/handlers/amqp/rabbitmq/RabbitMQMessagingTest.java

```diff
@@ -512,4 +516,41 @@ public void handleDelivery(String consumerTag,
         channel.close();
         conn.close();
     }
+
+    @Test(timeOut = 15_000)
+    public void testConsumeMoreThanInFlightSize() throws Exception {
+        Connection conn = getConnection("vhost1", true);
+        Channel channel = conn.createChannel();
+
+        String qu = randQuName();
+        channel.queueDeclare(qu, true, false, false, null);
+
+        long totalContentSize = conf.getManagedLedgerMaxReadsInFlightSizeInMB() * 1024 * 1024 * 2;
+        AtomicLong receiveContentSize = new AtomicLong(0);
+
+        CountDownLatch latch = new CountDownLatch(1);
+        channel.basicConsume(qu, new DefaultConsumer(channel) {
+            @Override
+            public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProperties properties,
+                                       byte[] body) throws IOException {
+                System.out.println("receive msg");
```

Comment:
It's better to use log instead of print.

Reply:
Oh, fixed.